### DISCUSSION TITLE
Nucleation at specific location in polycrystal

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#28451] [Q&A Modules: Phase field]
Hello,
I want to know if there is a way I can define (in my input file) that if certain variable reaches a particular value (eg. one of the order parameter say eta, when it is 0.5, or if a concentration variable, c, reaches say a value of 0.5) the nucleation takes place at those regions only. I am using DiscreteNucleationForce but this kernel wants me to act on the reserved order parameter. I am not able to see any module that let me define the condition. 
Looking forward.
Thank you
Ashish

**Comment by GiudGiud** [comment]
Hello

I think this is a use case the grain tracker.
https://mooseframework.inl.gov/source/postprocessors/GrainTracker.html

Guillaume

**Reply by ashishdhole** [reply]
Thank you
